apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -62,71 +76,62 @@ const triggerWebhook = async ({
   await Promise.all(promises);
 };
 
-const checkIfUserIsPartOfTheSameTeam = async (
-  teamId: number | undefined | null,
-  userId: number,
-  userEmail: string | undefined | null
-) => {
-  if (!teamId) return false;
-
-  const getUserQuery = () => {
-    if (!!userEmail) {
-      return {
-        OR: [
-          {
-            id: userId,
-          },
-          {
-            email: userEmail,
-          },
-        ],
-      };
-    } else {
-      return {
-        id: userId,
-      };
-    }
-  };
-
-  const team = await prisma.team.findFirst({
-    where: {
-      id: teamId,
-      members: {
-        some: {
-          user: getUserQuery(),
-        },
-      },
-    },
-  });
-
-  return !!team;
-};
+const testRequestSchema = z.object({
+  test: z.enum(["test"]),
+});
 
 async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
is the endpoint returning errors to the frontend ? if so it would be good to use defaultHandler / defaultResponder to make sure the errors are redacted and Prisma error stack is not returned to the frontend

Reply:
no this will not return any error to frontend. when recording access-link is ready. this route will get triggered